2-mandatory/1-writers.js

```diff
@@ -60,9 +60,13 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
-  // write your code to log all writers here
+  writers.forEach(function(e){
```

Comment:
I would rename `e` to `writer`. Try to make sure your variable names clearly indicate what they represent (like what Matilda mentioned above).

This also applies in other places where you have used `e`

Here's a link from the CYF styleguide that explains why: https://syllabus.codeyourfuture.io/guides/code-style-guide#avoid-short-names